spyderlib/qt/icon_manager.py

```diff
@@ -0,0 +1,218 @@
+import os
```

Comment:
I know, but I don't think it should belong to `spyderlib/qt` because it's our Qt shim.